qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_model.py

```diff
@@ -510,7 +511,22 @@ def test_calculate_axes_angles_uses_W_if_available(self):
         axes_angles = model.get_axes_angles(force_orthogonal=True)
         self.assertAlmostEqual(axes_angles[1, 2], np.pi / 2, delta=1e-10)
 
-    def test_calculate_axes_angles_uses_identity_if_W_unavailable(self):
+    def test_calculate_axes_angles_uses_W_if_available_MDHisto(self):
```

Comment:
So in this test it takes the values from the basis vectors of the workspace is that correct? If so this test would probably be better named something similar to 
`test_calculate_axes_angles_uses_basis_vectors_even_if_WMatrix_log_available_MDHisto`

Reply:
That's a good suggestion. I incorporated the name change.